google-cloud-automl/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
+### 0.2.0 / 2019-08-14
+
+
```

Comment:
Very minor nit, is it possible to only have one empty line instead of two?

Reply:
for sure, I'll put a test around this tomorrow when I come in.